### ISSUE TITLE
Ctrl Key gets stuck

### ISSUE THREAD

**Issue by aliqut** [end-4/dots-hyprland#665]
- I have read the Usage and Troubleshooting pages of the wiki: https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting/
- I have made sure that both my config and system packages are up to date
- Linux distro: OpenSUSE Tumbleweed

---
## The issue
Similar to https://github.com/end-4/dots-hyprland/issues/288#issuecomment-1964554707.
My CTRL key randomly gets stuck until it is pressed again, doesn't happen with any other keys. Killing fcitx5 fixes this, but I'm not sure what logs/configs to look through to resolve this.

**Comment by end-4** [comment]
have you tried using `fcitx5-configtool` then going to Global Options and removing cursed keybinds?

**Comment by end-4** [comment]
Hmm actually, I think I can confirm this does sometimes happen (not sure about killing to fix)
My laptop keyboard's Ctrl key isn't that great, so I've been thinking it's a hardware issue. But turns out it isn't, since I never have this problem when using Windows

**Comment by aliqut** [comment]
yeah, it's quite annoying. i'll look into the issue as well later.

**Comment by aliqut** [comment]
any update? i haven't been able to resolve the issue

**Comment by DaanVervacke** [comment]
I'm having the same issue.

**Comment by aliqut** [comment]
> I'm having the same issue.
@DaanVervacke 
remove fcitx5 from your hyprland execs

**Comment by daUnknownCoder** [comment]
oh wow i thought i was the only one with that issue, also i havent been getting that issue lately with this config (idr if this was the default @end-4 gave):

```zsh
## See https://fcitx-im.org/wiki/Using_Fcitx_5_on_Wayland
env = QT_IM_MODULE, fcitx
env = XMODIFIERS, @im=fcitx
# env = GTK_IM_MODULE, wayland   # Crashes electron apps in xwayland
# env = GTK_IM_MODULE, fcitx     # My Gtk apps no longer require this to work with fcitx5 hmm
env = SDL_IM_MODULE, fcitx
env = GLFW_IM_MODULE, ibus
env = INPUT_METHOD, fcitx

# Execute your favorite apps at launch
exec-once = dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP
```

version:
`hyprland-git 0.43.0.r90.e2426942-1`
`6.11.1-arch1-1`
`fcitx 4.2.9.9-2`

**Comment by daUnknownCoder** [comment]
> @DaanVervacke remove fcitx5 from your hyprland execs

that solved the issue for me too
